docs/source/models/concept_models/vivarium_nutrition_optimization/kids/concept_model.rst

```diff
@@ -486,6 +499,10 @@ Wave II:
 
 - It is possible we decide to exclude scenarios 13-18 from wave II, reducing the number of child scenarios from 19 to 13 and the total number of scenarios to 66/location for **198 scenarios**
 
+.. todo::
+
+  Update the below table to include targeted SQ-LNS scenarios. Then we will need to define a strategy for running the simulation since we will have multiple SQ-LNS targeting strategies and doing 3-4 complete runs might be infeasible.
+
```

Comment:
I think we can just "rerun" the scenarios that have coverage of targeted SQLNS with the different targeting algorithms rather than rerun _all_ scenarios, right? Sorry if I am misinterpreting your todo note here!

Reply:
Yep! Totally agree we don't need to. This will get replaced soon with the actual section so no changes but aligned.

Reply:
Great, sounds good